app/controllers/RSSFeeds.java

```
@@ -15,26 +17,28 @@
 public class RSSFeeds extends OBController {
 	
 	public static void addFeed(String url) {
-		if(!url.equals("")){
+		if(url != null && !url.trim().equals("")){
 			User u = user();
-			RSSFeed feed = new RSSFeed(u, url);
-			// Add Unique Constraint Check
-			feed.save();
+			RSSFeed feed = new RSSFeed(u, url.trim());
+      if (feed.is_valid) {
+        try {
+          feed.save();
+          renderJSON("1");
+        } catch (ConstraintViolationException e) {
```

Exception is not handled! Why do you even need to catch this exception. It must not happen.